libs/designsystem/src/lib/scss/_global-styles.scss

```diff
@@ -46,6 +46,12 @@ ion-modal.kirby-overlay {
     }
   }
 
+  &.interact-with-background {
+    &.kirby-modal.kirby-drawer {
+      --kirby-modal-padding-top: 0px;
+    }
+  }
+
```

Comment:
Consider moving to existing `&.kirby-drawer` section under `&.kirby-modal` section:
```
&.kirby-modal {
  ...
   &.kirby-drawer {
    ...
    &.interact-with-background {
      --kirby-modal-padding-top: 0px;
    }
  }
}
```